graylog2-server/src/main/java/org/graylog2/rest/resources/system/SessionsResource.java

```diff
@@ -126,10 +127,5 @@ public SessionResponse newSession(@Context ContainerRequestContext requestContex
     public void terminateSession(@ApiParam(name = "sessionId", required = true) @PathParam("sessionId") String sessionId) {
         final Subject subject = getSubject();
         securityManager.logout(subject);
-
-        final org.apache.shiro.session.Session session = subject.getSession(false);
```

Comment:
Why was this check removed? If at all, I'd move it before the call to `securityManager.logout(subject)` (because right now it effectively removes any active session the subject might have.